docs/build/build-with-thirdweb/one-click-deploy.md

```diff
@@ -56,37 +59,40 @@ By the end of this tutorial, you will:
 1. Clone the repository: 
  
    ```sh
-   git clone https://github.com/celo-org/one-click-quickstart
-   cd one-click-quickstart
+   git https://github.com/atejada/celo-one-click-deploy
+   cd celo-one-click-deploy
    ```
```

Comment:
- should be a Celo repo
- it's missing the "clone" in git clone

Reply:
Adding the "clone".

And yes, it should be a Celo repo, but I cannot create one ☹️ Could you create one so I can move it or transfer ownership or something?